ape_etherscan/verify.py

```diff
@@ -265,10 +265,12 @@ def attempt_verification(self):
             standard_json = json.dumps(standard_input_json, indent=2)
             logger.debug(f"{standard_json}\n")
 
+        # if there is a folder structure for source_id like xx/yy/zzz.sol
+        request_source_id = source_id.split("/")[-1]
```

Comment:
if you contract name different from the file name, it will still fail I think..

Should we do this instead:

```suggestion
        request_source_id = self._contract_type.name
```

Reply:
srry nvm